Microsoft.Toolkit.Services/PlatformSpecific/NETFramework/PopupWPF.xaml.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Forms;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace Microsoft.Toolkit.Services.PlatformSpecific.NetFramework
+{
+    /// <summary>
+    /// Interaction logic for PopupWPF.xaml
+    /// </summary>
+    public partial class PopupWPF : Window
+    {
+        private string initialHost;
+        private string callbackHost;
+
+        /// <summary>
+        /// Gets or sets the current URL before closing the form
+        /// </summary>
+        public Uri ActualUrl { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PopupWPF"/> class.
+        /// </summary>
+        /// <param name="callbackUrl">Uri callback url</param>
+        public PopupWPF(Uri callbackUrl)
+        {
+            InitializeComponent();
+
+            WebView1.NavigationStarting += (s, e) => WebViewNavigationStartingHandler(e.Uri);
```

Comment:
Needs -=.

Reply:
The WPF and Forms WebView control references the _WebView.Shared_ folder. So, when I try to use the same resource, inside this folder, it raises a namespace error. I tried to set an alias for one of these libraries, without success. Is there any way to use this resource inside both libraries?

Reply:
I don't think it's necessary to unregister the event here. WebView1 is part of the class and leak shouldn't be a problem